script.js

```diff
@@ -0,0 +1,32 @@
+let basic_monthly = document.getElementById("basic_monthly");
+let basic_annualy = document.getElementById("basic_annualy");
```

Comment:
As convention in JavaScript variables should be named in camelCase, please change here